### PR TITLE
add MountedBinaryResult

### PR DESCRIPTION
**Pull request by michaelvoronov**

### CODE REVIEW

File: crates/main/src/mounted_binary.rs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+use serde::Serialize;
+use serde::Deserialize;
+
+pub const SUCCESS_CODE: i32 = 0;
+
+/// Describes result of calling a CLI service.
+#[fce]
+#[derive(Clone, PartialEq, Default, Eq, Debug, Serialize, Deserialize)]
+pub struct Result {
```
Comment (folex) [inline crates/main/src/mounted_binary.rs:27]: Let's add something like
```
impl Result {
   fn into_std(self) -> Option<std::result::Result<String,String>>
}
```

that will check `ret_code` against `BINARY_SUCCESS_CODE` and return Ok or Err. 

P.S. maybe not `std::result::Result`, but a custom Result that is serialization friendly

Comment (michaelvoronov) [reply crates/main/src/mounted_binary.rs:27]: Why Option? I think it's better to make it `unsafe`.

Comment (folex) [reply crates/main/src/mounted_binary.rs:27]: Because it could be an invalid string

Comment (michaelvoronov) [reply crates/main/src/mounted_binary.rs:27]: Then it should be `Result<Option<String>, Option<String>>`

Comment (michaelvoronov) [reply crates/main/src/mounted_binary.rs:27]: Dicussed it and came to the idea of having two functions:
`pub fn into_std(self) -> Option<std::result::Result<String, String>>`
`pub fn as_std(&self) -> Option<std::result::Result<String, String>>`

**Review by folex — APPROVED** [review]

**Pull request merged**